app/Http/Controllers/FrontController.php

```diff
@@ -469,4 +469,64 @@ public function showAtlRamp() {
 	public function showCltRamp() {
         return view('site.ramp')->with('afld','CLT');
     }
+
+    public function pilotGuideAtl() {
+        $atc = ATC::get();
+        if($atc) {
+            $lcl_controllers = array();
+            $app_controllers = array();
+            $ctr_controllers = array();
+            foreach($atc as $a) {
+                $field = substr($a->position, 0, 3);
+                $position = substr($a->position, -3);
+                if(($field == 'ATL')||($field == 'ZTL')) {
+                    if($position == 'TWR' || $position == 'GND' || $position == 'DEL') {
+                        $lcl_controllers[] = $a;
+                    } elseif($position == 'APP' || $position == 'DEP') {
+                        $app_controllers[] = $a;
+                    } elseif($position == 'CTR') {
+                        $ctr_controllers[] = $a;
+                    }
+                } 
+            }
+        }
+        if(count($lcl_controllers) < 1) {
+            $lcl_controllers = $app_controllers;
+            if(count($lcl_controllers) < 1) {
+                $lcl_controllers = $ctr_controllers;
+            }
+        }
+        $last_update = ControllerLogUpdate::orderBy('id', 'desc')->first();
+        $controllers_update = substr($last_update->created_at, -8, 5);
+
+        $client = new Client(['http_errors' => false]);
+        $icao_id = 'KATL';
+        $res = $client->request('GET', 'https://api.aviationapi.com/v1/charts?apt=' . $icao_id);
+        $status = $res->getStatusCode();
+        $diag = $aaup = '#';
+        if($status == 404) {
+            $charts = null;
```

Comment:
Change to `if($status != 200)` and then this will catch any time it's not successful. In the current state, if the API returns a 500 status code or something like that, it'll still try to pull the data that it can't get and it'll probably error and break our website.

Reply:
Makes sense - easy fix.

Reply:
Done, per your suggestion.